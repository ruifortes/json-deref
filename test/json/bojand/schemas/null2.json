 {
   "description": "Just a basic schema.",
   "title": "Basic Object",
   "type": "object",
   "properties": {
     "id": {
       "$ref": "db://custom.json#/id.json"
     }
   },
   "additionalProperties": false
 }
