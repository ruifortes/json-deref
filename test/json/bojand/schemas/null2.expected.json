{
  "description": "Just a basic schema.",
  "title": "Basic Object",
  "type": "object",
  "properties": {
    "id": {
      "type": "string"
    }
  },
  "additionalProperties": false
}
